antsibull/data/docsite/plugin.rst.j2

```diff
@@ -409,7 +438,26 @@ Status
 
 .. Deprecated note
 
-- This @{ plugin_type }@ will be removed in version @{ doc['deprecated']['removed_in'] | default('') | string | rst_ify }@. *[deprecated]*
+{% if (doc['deprecated']['removed_from_collection'] or collection_name) != collection_name %}
+{%   if 'removed_at_date' in doc['deprecated'] %}
```

Comment:
I think that these keys will now always be present in `doc['deprecated']` (that's how I try to write the schemas) so we need to check the value instead?

Reply:
Yep.